CHANGELOG.rst

```diff
@@ -13,6 +13,17 @@ Added
 * Add support for utf-8 / unicode characters in the pack config files. (improvement) #3980 #3989
 
   Contributed by @sumkire.
+* Add support for new optional ``content_version`` runner parameter to the Python and Local Shell
+  Script runner. This parameter can contain a git commit hash / tag / branch from a pack git
+  repository and runner will ensure this revision of the pack content (Python action / local shell
+  script action) is used for a particular action execution.
```

Comment:
How come only python and local shell runners are supported?  From what i review of GitWorktreeActionRunner, it's overriding the entry point. So shouldn't this work for all actions with an entry point?  Please explain why this wouldn't work for other runners.

Reply:
The feature makes only makes sense / applies to runners which use local script files (entry point) from disk. It allows users to use previous version of that entry_point script file based on the value of `content_version` value.

Those are mainly Python and local script runner.

It makes no sense for runners such as local command runner and remote command runner, http runner, etc. because in those cases there is no local script file and all the commands / parameters are provided during invocation.

Reply:
How about action chain and mistral? They have a WF definition file on disk.

Reply:
Yeah, I was thinking about Mistral, but I believe that would also require work on the Mistral side - to make it use a different directory with workflow definition when `content_version` parameter is specified.

Reply:
It shouldn't require a lot of work since you already got this figured out. The WF definition for both Action Chain and Mistral are specified in the entry point.  Should just make those runners inherit from GitWorktreeActionRunner?  Need to write unit tests for them.